user_guide/src/examples/expressions/with_column_context_1.py

```diff
@@ -1,7 +1,7 @@
 from .dataset import df
 import polars as pl
 
-df = df.with_columns(
+out = df.with_columns(
```

Comment:
We import these in `expressions/__main__.py` so we need to change the name there as well.